packages/ui/core-components/src/lib/unsorted/viz/table/_DataTable.svelte

```diff
@@ -44,9 +44,18 @@
 	$: rowNumbers = rowNumbers === 'true' || rowNumbers === true;
 
 	// Sort props
-	export let sortBy = undefined;
-	export let sortAsc = false;
-	$: sortAsc = sortAsc === 'true' || sortAsc === true;
+	export let sort = undefined;
+	let sortBy = undefined;
+	let sortAsc = undefined;
+	$: if (sort) {
+		const [column, direction] = sort.split(' ');
```

Comment:
this feels fragile but fully be sure why.

is this allowed?
```svelte
<DataTable 
  data={query}
  sort="column_one, column_two desc"
/>
```

Reply:
Wasn't contemplated - I'll take a look

Reply:
This isn't supported - only allows for one column in this version.

I had the same thought about this seeming fragile. If there's a mistake, at the moment it fails silently. 

Might be worth including error messages for common scenarios people might run into. Some that come to mind:
- Attempting to use multiple columns in sort
- Misspelling column name
- Misspelling direction asc or desc
- Not wrapping column + direction in quotes in the prop

Any others?